packages/shared/components/inputs/Pin.svelte

```diff
@@ -53,27 +53,26 @@
         value = inputs.join('')
     }
 
-    const changeHandler = function (e, i) {
+    function changeHandler(event: KeyboardEvent): void {
         const regex = new RegExp(/^\d+$/)
-
-        if (e.keyCode === KEYBOARD.BACKSPACE) {
+        if (event.key === KEYBOARD.BACKSPACE) {
             handleBackspace()
-        } else if (e.keyCode === KEYBOARD.ENTER) {
+        } else if (event.key === KEYBOARD.ENTER) {
             if (validatePinFormat(inputs.join(''))) {
                 dispatch('submit')
             }
-        } else if (e.keyCode === KEYBOARD.TAB) {
+        } else if (event.key === KEYBOARD.TAB) {
             // Do default tab handling by focusing the root
             // container and allow default processing to happen
             root.focus()
```

Comment:
Should this be changed in a separate PR? Good call on switching to `key` btw — I had been meaning to change that and then I forgot 😅

Reply:
I think it's fine like this. Imo it's not worth of creating a separate PR. There are 2 other files which have `keyCode`, I'll update them aswell and put it in the change description of this PR.

Reply:
IMO it should be in a separate PR because it can affect the functionality of the PIN component (making it potentially user-facing, and no longer considered a chore). We've seen reports of issues with other keyboard layouts (https://github.com/iotaledger/firefly/issues/1934), so this change should be tested with multiple keyboard layouts and OSes. I'm happy to help with that btw, but it may take a bit more time and could delay this PR if it's included in it

Reply:
IMO it should be in a separate PR because it can affect the functionality of the PIN component (making it potentially user-facing, and no longer considered a chore). We've seen reports of issues with other keyboard layouts (https://github.com/iotaledger/firefly/issues/1934), so this change should be tested with multiple keyboard layouts and OSes. I'm happy to help with that btw, but it may take a bit more time and could delay this PR if it's included in it

Reply:
I cherry-picked the two relevant commits and opened a PR here: https://github.com/iotaledger/firefly/pull/3360.
Probably need to rebase once the PR has been tested. Can you test the different layouts and OS'es?